src/core/packages/chrome/browser-internal/src/project_navigation/README.md

```diff
@@ -0,0 +1,112 @@
+# Project Navigation Service
```

Comment:
Will this be available on the developer documentation site, or is that not the goal? If we want it there, I think we need a header for the developer documentation here.

Reply:
I guess it shouldn't. This is just internal implementation detail.